.travis.yml

```diff
@@ -5,30 +5,29 @@ language: bash
 services:
   - docker
 
-before_install:
-  - docker-compose -v
-
-install:
+before_script:
+  - export APP_IMAGE=keboola-component
+  - docker -v
+  - docker build -t $APP_IMAGE .
   - docker-compose build
   - docker-compose run --rm app composer ci
 
-script:
+  # push test image to ECR
   - docker pull quay.io/keboola/developer-portal-cli-v2:latest
-  - export REPOSITORY=`docker run --rm -e KBC_DEVELOPERPORTAL_USERNAME -e KBC_DEVELOPERPORTAL_PASSWORD
-    quay.io/keboola/developer-portal-cli-v2:latest ecr:get-repository
-    keboola keboola.ex-db-snowflake`
-  - docker tag keboola/ex-db-snowflake:latest $REPOSITORY:master
-  - eval $(docker run --rm -e KBC_DEVELOPERPORTAL_USERNAME -e KBC_DEVELOPERPORTAL_PASSWORD
-    quay.io/keboola/developer-portal-cli-v2:latest ecr:get-login
-    keboola keboola.ex-db-snowflake)
-  - docker push $REPOSITORY:master
+  - export REPOSITORY=`docker run --rm -e KBC_DEVELOPERPORTAL_USERNAME -e KBC_DEVELOPERPORTAL_PASSWORD -e KBC_DEVELOPERPORTAL_URL quay.io/keboola/developer-portal-cli-v2:latest ecr:get-repository $KBC_DEVELOPERPORTAL_VENDOR $KBC_DEVELOPERPORTAL_APP`
+  - docker tag $APP_IMAGE:latest $REPOSITORY:test
+  - eval $(docker run --rm -e KBC_DEVELOPERPORTAL_USERNAME -e KBC_DEVELOPERPORTAL_PASSWORD -e KBC_DEVELOPERPORTAL_URL quay.io/keboola/developer-portal-cli-v2:latest ecr:get-login $KBC_DEVELOPERPORTAL_VENDOR $KBC_DEVELOPERPORTAL_APP)
+  - docker push $REPOSITORY:test
```

Comment:
neřeší to všechno deploy script?

Reply:
toto som skopiroval z generatora, ... musi tam byt sptavny tag `test` v starom bolo `master`:

Porovnaj: https://github.com/keboola/component-generator/blob/master/templates/php-component/.travis.yml

Reply:
cc @ondrajodas

Reply:
jj jasně to je test tag 👍